modules/nf-core/samtools/cramsize/main.nf

```diff
@@ -0,0 +1,35 @@
+process SAMTOOLS_CRAMSIZE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::samtools=1.17"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.17--h00cdaf9_0':
+        'biocontainers/samtools:1.17--h00cdaf9_0' }"
+
+    input:
+    tuple val(meta), path(cram)
+
+    output:
+    tuple val(meta), path("*.size"), emit: size
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
+    """
+    samtools \\
+        cram-size \\
+        $args \\
+        -o ${cram.baseName}.size \\
+        $cram
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        : \$(echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//' ))
+    END_VERSIONS
+    """
+}
```

Comment:
Can you add a stub section?

Reply:
Thanks for all your advice @nvnieuwk ! I have just added a stub section and a test.yml file. I will be very grateful if you can review it, please. 

Thanks a lot in advance!